debvader/metrics.py

```diff
@@ -0,0 +1,10 @@
+import numpy as np
+
+
+def mse(img1, img2):
+    return np.mean(np.square(img1 - img2))
+
+
+# Define the loss as the log likelihood of the distribution on the image pixels
+def vae_loss(x, x_decoded_mean):
+    return -x_decoded_mean.log_prob(x)
```

Comment:
Two things : 
- now that the loss has been separated from the code it needs to be more explicit and renaming of the second argument
- comment => docstring

+ its the **negative** log likelihood of the distribution **of** the image pixels